pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -134,6 +149,30 @@ func calculateCPUMaximum(cpuLimit *resource.Quantity, cpuCount int64) int64 {
 }
 
 func toKubeContainerResources(statusResources *runtimeapi.ContainerResources) *kubecontainer.ContainerResources {
-	//TODO: Add windows support
-	return nil
+	var cStatusResources *kubecontainer.ContainerResources
+	runtimeStatusResources := statusResources.GetWindows()
+	if runtimeStatusResources != nil {
+		var memLimit, cpuLimit *resource.Quantity
+
+		// Starting from this formula, mentioned in calculateWindowsResources function:
+		// cpuMaximum := 10000 * cpuLimit.MilliValue() / int64(runtime.NumCPU()) / 1000
```

Comment:
this formula was simplified on line 140 above in `calculateCPUMaximum`.  Can we update it to match to avoid confusion later?

I am having a hard time from just this code (with little context to be fair) as to what we are doing here.  looks like we are doing something similar but not quite the same as `calculateCPUMaximum`?

Reply:
can we re-use the function or is there subtle differences between the two?

Reply:
There is no cpuRequest for Windows, which is why I could not reuse that function

Reply:
oh I see this is the reverse of that function.  Can we update the formula in the comment on this line as it doesn't match what is in `calculateWindowsResources`?

I think this was what initially had me confused.  Bonus points for explicitly mentioning this is the reverse of that function.

Reply:
Modified the function to match that one: `cpuLimitValue := runtimeStatusResources.CpuMaximum * int64(winstats.ProcessorCount()) / 10`

Reply:
Can you update the comment? the comment doesn't match the current code and it is confusing

Reply:
Sure, changed it to point to the calculateCPUMaximum function